eclair-core/src/main/scala/fr/acinq/eclair/NodeParams.scala

```diff
@@ -93,18 +93,28 @@ object NodeParams {
 
   object ELECTRUM extends WatcherType
 
+  def checkConfiguration(config: Config): Config = {
+    // this check is needed because somme keys were moved to a new section in v0.3.2
+    if (config.hasPath("eclair.default-feerates") || config.hasPath("eclair.max-feerate-mismatch") || config.hasPath("update-fee_min-diff-ratio")) {
```

Comment:
Shouldn't `config.hasPath("update-fee_min-diff-ratio")` be `config.hasPath("eclair.update-fee_min-diff-ratio")`? I see it inside the `eclair` block in [0.3.1](https://github.com/ACINQ/eclair/blob/v0.3.1/eclair-core/src/main/resources/reference.conf#L81)

Reply:
Good catch thanks!